.github/workflows/nightly_tests.yml

```diff
@@ -124,6 +110,206 @@ jobs:
           pip install slack_sdk tabulate
           python scripts/log_reports.py >> $GITHUB_STEP_SUMMARY
 
+  run_nightly_tests_for_other_torch_modules:
+    name: Torch Non-Pipelines CUDA Nightly Tests
```

Comment:
I think we can just call it "Torch CUDA Nightly Tests"

Reply:
I wanted to specifically distinguish between pipeline and non-pipeline tests for convenience.